pkgs/by-name/xe/xeve/package.nix

```diff
@@ -17,18 +18,61 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-8jXntm/yFme9ZPImdW54jAr11hEsU1K+N5/7RLmITPs=";
   };
 
+  patches = lib.optionals (! lib.versionOlder "0.5.0" finalAttrs.version)
+    (builtins.map fetchpatch2 [
+      { url = "https://github.com/mpeg5/xeve/commit/07a6f2a6d13dfaa0f73c3752f8cd802c251d8252.patch?full_index";
+        hash = "sha256-Dul3PESwGbeYqId9kzRcaLhDdts00JiqUKeLY5oDB24=";
+      }
+      { url = "https://github.com/mpeg5/xeve/commit/0a0f3bd397161253b606bdbeaa518fbe019d24e1.patch?full_index";
+        hash = "sha256-ByJOnGbymUTZOgSL59yPjm/oeBtcPivrHGc3LEt4/tE=";
+      }
+      { url = "https://github.com/mpeg5/xeve/commit/e029f1619ecedbda152b8680641fa10eea9eeace.patch?full_index";
+        hash = "sha256-kUhJ87m5AlNY+KlI8NF+SWZbDOPsAiPdWoyaBJuiny0=";
+      }
+    ] ++ [
+      # Backport to 0.5.0 of upstream patch c564ac77c103dbba472df3e13f4733691fd499ed
+      ./0001-CMakeLists.txt-Disable-static-linking-on-Darwin.patch
+    ]) ++ [
+    # Rejected upstream, can be dropped when a fix for
+    # https://github.com/mpeg5/xeve/pull/123 is in a version bump.
+    ./0002-sse2neon-Cast-to-variable-type.patch
+  ];
+
+
   postPatch = ''
     echo v$version > version.txt
   '';
 
   nativeBuildInputs = [ cmake ];
 
+  cmakeFlags = let inherit (lib) cmakeBool cmakeFeature optional;
+                   inherit (stdenv.hostPlatform) isAarch64 isDarwin;
+                in optional isAarch64 (cmakeBool "ARM" true)
+                ++ optional isDarwin
+                            (cmakeFeature "CMAKE_SYSTEM_NAME" "Darwin");
+
+  env.NIX_CFLAGS_COMPILE = builtins.toString (
+    builtins.map (w: "-Wno-error=" + w) [
```

Comment:
Same as above comment.

Is `-lm` no longer required?

Reply:
On linux this was requires so that ffmpeg could find it, no idea if any of @toonn's patches fixed the need for that or not.

Reply:
The CmakeLists.txt includes it already so I'd be surprised if it's required on Linux?

Reply:
I guess it still is?